.github/CODEOWNERS

```diff
@@ -1115,11 +1115,17 @@
 #PRLabel: %CodeGen %Client
 /eng/packages/http-client-csharp/                                  @JoshLove-msft @m-nash @jorgerangel-msft @live1206 @ArcturusZhang
 
+#PRLabel: %CodeGen %Mgmt
+/eng/packages/http-client-csharp-mgmt/                                  @JoshLove-msft @m-nash @jorgerangel-msft @live1206 @ArcturusZhang
+
 # ServiceLabel: %CodeGen
 # AzureSdkOwners:                                                  @JoshLove-msft
 
 #PRLabel: %CodeGen %Client
 /eng/packages/http-client-csharp/generator/Azure.Generator/        @JoshLove-msft @m-nash @jorgerangel-msft @jsquire @live1206 @ArcturusZhang
```

Comment:
This block is redundant to the one on L1115.   Please remove this one and add me to the higher one.

Reply:
removed